php/class-photonfill.php

```diff
@@ -454,42 +479,52 @@ public function create_url_image_object( $img_url, $current_size, $args = array(
 						$default = ( ! empty( $img_size['default'] ) ) ? true : false;
 						$current_w = empty( $img_size['width'] ) ? 0 : $img_size['width'];
 						$current_h = empty( $img_size['height'] ) ? 0 : $img_size['height'];
-						$transform_args = array(
+						$transform_args = apply_filters( 'photonfill_pre_transform_args', array(
 							'callback' => ( isset( $img_size['callback'] ) ) ? $img_size['callback'] : null,
 							'crop' => ( isset( $img_size['crop'] ) ) ? $img_size['crop'] : true,
 							'breakpoint' => $breakpoint,
 							'image_size' => $current_size,
 							'width' => $current_w,
 							'height' => $current_h,
 							'quality' => ( isset( $img_size['quality'] ) ) ? $img_size['quality'] : null,
-						);
+						), 'external_url', $current_size, $args );
```

Comment:
`'external_url'` seems to be used here, and again below in a couple places like...